### PR TITLE
Open a new PopUp window 

### PR DESCRIPTION
**Pull request by anas599**
- Open a new pop-up window when clicking on see project button
- create an array, which contains an object with all the project details to be loaded dynamically

### CODE REVIEW

File: Assets/popUpObject copy [hunk 1]
```diff
@@ -0,0 +1,131 @@
+const width = window.innerWidth;
+/* eslint-disable no-unused-vars */
+const projectsInfo = [
+  {
+    name: 'Multi-Post Stories Gain+Glory',
```
Comment (Kossi-stack) [inline Assets/popUpObject copy:5]: - [ ] Kindly increase the size of the card as well as the text description.
Screenshot 2022-12-23 at 08 19 32

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @anas599,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻️

 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: Assets/popUpObject.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
-/* eslint-disable no-constant-condition */
 /* eslint-disable no-unused-vars */
 const projectsInfo = [
   {
     name: 'Multi-Post Stories Gain+Glory',
+    namePop: 'Keeping track of hundreds of components',
```
Comment (roshan-bajgain) [inline Assets/popUpObject.js:5]: - [x] [Optional] You should stick to the design as much as possible like color, font, and spacing. Please make your `x` buttons similar to the design and also stick to the design for buttons and use the same color for text. Please see the image below for reference
### Your design
your
### Figma design
fimga

**Review by roshan-bajgain — APPROVED** [review]
Hi @anas599, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

## Optional suggestions
- [ ] I cannot leave comments there so I am leaving them here, there are many commented codes I saw in the HTML file please remove them for clean work.
- [ ] I notice that this is a pair programming project and you did it alone, please keep this in mind for next time. Always do this with your partner if the project is pair programming this will help you when you work in a remote environment.
__Check comment under the review__

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: Assets/popUpObject.js (hunk 2 above)
Comment (anas599) [reply Assets/popUpObject.js:5]: Hi Roshan, thank you for stating the issues 
My coding partner did in fact contribute to my project but I'm not sure why it is not showing in this pull request

**Comment by roshan-bajgain** [comment]
@anas599, This is the commit history of your project and your partner did not commit at all or you did not push the code. please check it
### Screenshot
Screenshot 2022-12-23 230132

File: Assets/popUpObject.js (hunk 2 above)
Comment (anas599) [reply Assets/popUpObject.js:5]: Screenshot 2022-12-23 at 8 23 51 PM

Hello Roshan,
Kindly check the comments a few days ago as shown in screenshot,
I appreciate your help.

**Pull request merged**